todo/src/actions/index.js

```diff
@@ -0,0 +1,26 @@
+export const ADD_TODO = 'ADD_TODO';
+export const UPDATE_COMPLETED = 'UPDATE_COMPLETED';
+export const DELETE_ITEM = 'DELETE_ITEM';
+
+let nextID = 0;
+export const add_todo = todo => {
+  const payload = { id: nextID++, todo: todo, completed: false };
+  return {
+    type: 'ADD_TODO',
+    payload: payload,
```

Comment:
if the key that you're defining has a variable of the same name you only need to list it once
```
{
  type: 'ADD_TODO',
  payload,
}
```